stylix/testbed/modules/application.nix

```
@@ -26,6 +31,11 @@ in
               used may change in the future.
             '';
           };
+          graphicalEnvironment = lib.mkOption {
+            type = lib.types.enum availableGraphicalEnvironments;
```

What I meant with https://github.com/nix-community/stylix/pull/1222#discussion_r2202860984 and https://github.com/nix-community/stylix/pull/1222#discussion_r2202863005 is to inline the `availableGraphicalEnvironments` identifier. Not to [remove and inline the called `available-graphical-environments.nix` file](https://github.com/nix-community/stylix/compare/5a1eebf1bae4be94b92cce7adbf184bae5113a5b..0c5c1f76369dde10c3b10b08621847c69347aad3). I would have been fine with duplicating this small amount of code, but since you already proposed it, you might as well keep it:

```suggestion
            type = lib.types.enum (
              import ../available-graphical-environments.nix { inherit lib; }
            );
```